packages/site-server/package.json

```diff
@@ -9,13 +9,15 @@
   "scripts": {
     "build": "wireit",
     "check": "wireit",
-    "serve:dev": "wireit"
+    "serve:dev": "wireit",
+    "serve:prod": "wireit"
   },
   "wireit": {
     "build": {
       "command": "find src -name \"*.ts\" | xargs esbuild --format=esm --target=es2022 --color --outdir=lib",
       "dependencies": [
-        "../custom-elements-manifest-tools:build"
+        "../custom-elements-manifest-tools:build",
+        "../site-client:build"
```

Comment:
I think since we don't have "soft" dependencies in wireit yet, I would move this to the `serve:prod` and `serve:dev` scripts. Otherwise in service mode, we'll restart the server whenever the assets change, which we don't want.

Reply:
We do need that now because these are used in the server.